Added missing whitespace after numbered emojis in all languages [Fixe… (#12179)
ethereum/ethereum-org-website: public/content/contributing/design/index.md

## How to contribute?

> ### <Emoji text=":one:" size={1} me="2" /> Provide feedback on early design prototypes {#design-critique}

Reviewer: ```suggestion
### <Emoji text=":one:" size={1} /> &nbsp;Provide feedback on early design prototypes {#design-critique}
```

We sometimes need a help testing our raw ideas. This is a great way how to contribute without any technical knowledge.

1. The design team will share a mockup design on [Discord](https://discord.com/invite/ethereum-org) and on [GitHub](https://github.com/ethereum/ethereum-org-website/labels/design%20required%20%F0%9F%8E%A8).
2. You will be guided through the designs to provide feedback via comments function.
3. The outcome will be shared in the GitHub issue and then closed by the team.

> ### <Emoji text=":two:" size={1} me="2" /> Participate in survey research {#answer-surveys}

Reviewer: ```suggestion
### <Emoji text=":two:" size={1} /> &nbsp;Participate in survey research {#answer-surveys}
```

Provide feedback on our website by:

1. Visiting ethereum.org and reading through pages.
2. Clicking on the feedback widget at the bottom right corner and answering design and content-related questions.
3. Focus on the free format questions.

> ### <Emoji text=":three:" size={1} me="2" /> Find design related issues on the website and report them {#report-design-issues}

Reviewer: ```suggestion
### <Emoji text=":three:" size={1} /> &nbsp;Find design related issues on the website and report them {#report-design-issues}
```

Ethereum.org is a fast growing website with many features and content. Some of the UI can easily become obsolete or could be improved. If you encounter any such instance, please report it so that it gets our attention.

1. Go through the website and pay attention to its design.
2. Take screenshots and notes if you see any visual or UX issues.
3. Report the found issues using a [bug report](https://github.com/ethereum/ethereum-org-website/issues/new/choose).

> ### <Emoji text=":four:" size={1} me="2" /> Propose design changes {#propose-design-changes}

Reviewer: ```suggestion
### <Emoji text=":four:" size={1} /> &nbsp;Propose design changes {#propose-design-changes}
```

3. Propose the solution in the corresponding GitHub issue or [create a new one.](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A&template=feature_request.yaml&title=Feature+request)
4. Wait for the design team to review.

> ### <Emoji text=":five:" size={1} me="2" /> Build Design System together {#Contribute-to-design-system}

Reviewer: ```suggestion
### <Emoji text=":five:" size={1} /> &nbsp;Build Design System together {#Contribute-to-design-system}
```

5. The design team will review.
6. The Design team will incorporate the changes in the main file and publish the file to the community.

> ### <Emoji text=":six:" size={1} me="2" /> Write design-related content on the website {#write-design-articles}

Reviewer: ```suggestion
### <Emoji text=":six:" size={1} /> &nbsp;Write design-related content on the website {#write-design-articles}
```

4. Once approved, write the content.
5. Submit it in the corresponding GH issue.

> ### <Emoji text=":seven:" size={1} me="2" /> Draw new illustrations {#prepare-illustrations}

Reviewer: ```suggestion
### <Emoji text=":seven:" size={1} /> &nbsp;Draw new illustrations {#prepare-illustrations}
```
